src/authResolver.ts

```diff
@@ -95,7 +95,7 @@ export class RemoteSSHResolver implements vscode.RemoteAuthorityResolver, vscode
                 const sshconfig = await SSHConfiguration.loadFromFS();
                 const sshHostConfig = sshconfig.getHostConfiguration(sshDest.hostname);
                 const sshHostName = sshHostConfig['HostName'] ? sshHostConfig['HostName'].replace('%h', sshDest.hostname) : sshDest.hostname;
-                const sshUser = sshHostConfig['User'] || sshDest.user || '';
+                const sshUser = sshHostConfig['User'] || sshDest.user || process.env.USER || '';
```

Comment:
Thanks @lex-ibm for the PR, just one thing, what about `os.userInfo` so it works on windows too

Reply:
@jeanp413 you are right! I've changed it to use `os.userInfo`.